### DISCUSSION TITLE
Onboarding Clarity

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/caMicroscope#671] [General]
Hello all,

It seems that some people are having some problems or have unanswered questions about how to contribute to caMicroscope (http://camicroscope.org/contributing.html)

Please let us know if anything seems unclear, or if we can guide people to their first PR!

**Comment by katg999** [comment]
Hello, 
       This is BrianGilbert  with specialised interest and skillset in machine learning and have picked interest in the Image Classification Using Foundational Models but wanted some clarification on the project requirements like integration  of this method as a set of software components  and  also how can i achieve my first PR regarding the selected idea above.  Thanks
